### DISCUSSION TITLE
Procedural Macros?

### DISCUSSION THREAD

**Discussion by joehillen** [hylo-lang/hylo#1518] [Q&A]
Any plans to support Rust-style Procedural Marcos?

**Comment by dabrahams** [comment]
No plans as of now.  We're a fundamentals-first language, and we have a lot of those to put in place before we think about anything like that.  We're not necessarily macro-hostile, but we are at least macro-suspicious.  If you think this is an essential feature, I suggest bringing this issue up again after 1.0 is released.
